app/transport/rest/user_test.go

```diff
@@ -202,3 +205,91 @@ func TestUserHandler_GetAllUsers(t *testing.T) {
 		})
 	}
 }
+
+func TestUserHandler_GetUserByID(t *testing.T) {
+	logger, err := lgr.New(lgr.InfoLevel)
+	if err != nil {
+		t.Fatal("cannot initialize logger")
+	}
+
+	tests := map[string]struct {
+		id                    string
+		usecaseBuilder        func(ctrl *gomock.Controller) UserUsecase
+		expectedStatusCode    int
+		expectedResponsetBody string
+	}{
+		"success": {
+			id: "91e3dcf7-34a6-4646-bd37-383cc949da93",
+			usecaseBuilder: func(ctrl *gomock.Controller) UserUsecase {
+				mock := NewMockUserUsecase(ctrl)
+
+				mock.EXPECT().GetByID("91e3dcf7-34a6-4646-bd37-383cc949da93").Return(&entities.User{
```

Comment:
Entity `user` has the field `password` and I think that the mock func should return that field too. The reason of the test is to ensure that the handle will not return the password even if the mock func will return it

Reply:
Nice catch. Fixed it